src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -69,8 +69,20 @@
                     i18n-placeholder
                     id="token"
                     formControlName="token"
-                    readonly>
-          </textarea>
+                    readonly
+                    #token></textarea>
+        </div>
+
+        <div class="btn-group mb-3"
+             role="group"
+             aria-label="Copy To Clipboard">
+          <button class="btn btn-primary"
+                  title="Copy to Clipboard"
+                  i18n-title
+                  (click)="copyToClipBoard(token)"> Copy to Clipboard
+            <i class="fa fa-files-o"
```

Comment:
I don't remember that we display text and icon in buttons at the same time. So please remove the icon.

Reply:
done